### PR TITLE
docstrings in inplace_decrement #2908

### PR DESCRIPTION
**Pull request by edaehn**
Added docstring examples, checked arguments, fixed inplace_decrement #2908

### CODE REVIEW

**Comment by edaehn** [comment]
closes #2908

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -2262,10 +2262,28 @@ def set_queue_timeout(timeout):
 
 
 def queue_timeout():
```
Comment (rush2406) [inline ivy/functional/ivy/general.py:2264]: This function need not be formatted in this PR, can you please remove the changes as it's causing `docstrings` to fail

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @edaehn 
Thanks for contributing! You would also need to add static and instance methods for arrays and containers along with some examples. You can follow these for reference [container](https://github.com/unifyai/ivy/blob/50a09216faaf254f4c7e3a8e758883e1fa5be297/ivy/container/general.py#L84), [array](https://github.com/unifyai/ivy/blob/50a09216faaf254f4c7e3a8e758883e1fa5be297/ivy/array/general.py#L379)

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (edaehn) [reply ivy/functional/ivy/general.py:2264]: Hi @rush2406, thanks for your message. I will add these methods soon.

File: ivy/array/general.py [hunk 2]
```diff
@@ -327,6 +327,42 @@ def to_list(self: ivy.Array):
         """
         return ivy.to_list(self)
 
+    def inplace_decrement(
+        self: Union[ivy.Array, ivy.NativeArray], val: Union[ivy.Array, ivy.NativeArray]
+    ) -> ivy.Array:
+        """ivy.Array instance method variant of ivy.inplace_decrement. This method simply
```
Comment (ankitdsi2010) [inline ivy/array/general.py:333]: Refer [lint check](https://github.com/unifyai/ivy/runs/7839381090?check_suite_focus=true#step:5:7) and fix it.

**Review by ankitdsi2010 — CHANGES_REQUESTED** [review]
fix the failing tests in the run-docstring-tests

File: ivy/array/general.py (hunk 2 above)
Comment (edaehn) [reply ivy/array/general.py]: Hi @ankitdsi2010, thanks for your comment. Lint tests are passing OK now. Best regards, Elena.

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (edaehn) [reply ivy/functional/ivy/general.py]: Hi @rush2406 and @ankitdsi2010 , 

thank you very much for your comments. I made the changes. However, the docstring tests fail due to some issues in /functional/ivy/activations.py (relu(), line 55). To test, I run in Docker schell: 

>>> x = ivy.array([1.5, 0.7, -2.4])
>>> y = ivy.zeros(3)

It results in:

  File "/ivy/ivy/functional/ivy/creation.py", line 185, in zeros
    return current_backend().zeros(shape, dtype=dtype, device=device, out=out)
TypeError: zeros() got an unexpected keyword argument 'out'.

It seems to be unrelated to the inplace_decrement(), and it is OK to do changes in the  relu() in this PR. Please advice. 

Thank you very much. 

Best regards, Elena.

**Review by rush2406 — APPROVED** [review]

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (rush2406) [reply ivy/functional/ivy/general.py]: @edaehn It's fine, you need not worry about the failures from other functions

Comment (edaehn) [reply ivy/functional/ivy/general.py]: Great! Thank you very much @rush2406

**Review by ankitdsi2010 — APPROVED** [review]

**Comment by ankitdsi2010** [comment]
Thanks for your contribution

**Pull request merged**